idd/Energy+.idd.in

```diff
@@ -48608,6 +48608,61 @@ ZoneHVAC:AirDistributionUnit,
        \type object-list
        \object-list DesignSpecificationAirTerminalSizingName
 
+ZoneHVAC:ExhaustControl,
+       \memo Defines a controlled exhaust flow from a zone which feeds into 
+       \memo an AirloopHVAC:Exhaust system inlet.
```

Comment:
an AirloopHVAC:ZoneMixer inlet which is part of an AirloopHVAC;Exhuast system.

Reply:
Texts corrected.